tests/test_gather_financial_statement_company_compare.py

```diff
@@ -0,0 +1,70 @@
+from pyntrinio.pyntrinio import gather_financial_statement_company_compare
+"""
+This script tests the gather_financial_statement_company_compare functions in the pyntrinio module.
+"""
+
+
+def test_type_input():
+    '''
+    Tests if the types of the inputs are the right ones
+    ''' 
+    api_key = 'OmEzNGY3MGEwMDIwZGM5Y2UxNDZhNzUzMTgzYTJiNWI2'
+    ticker = ['AAPL', 'CSCO']
+    statement = 'income_statement'
+    year = '2014'
+    period = 'Q1'
+    
+    #Check that when the api_key is not a string, it returns an error
+    with raises(TypeError):
+        gather_financial_statement_company_compare(['la'], ticker, statement, year, period)
+    
+    #Check that when the statement is not a string, it returns an error
+    with raises(TypeError):
```

Comment:
Can we make this specific so it raises an error when none of the valid input types #42 ?

Reply:
Sorry for not being more specific - I had been referring to the statement type: income_statement, balance_sheet_statement, cash_flow_statement

I don't want to accept the changes listed because in the non-Free access versions, you can use any ticker, and this would prevent someone from doing that.